index.js

```diff
@@ -5,7 +5,18 @@ const _ = require('lodash');
 class Config {
     constructor() {
         if (!Config.data) {
-            console.log("Using config for Clowder at: " + process.env.ACG_CONFIG)
+            if(!IsClowderEnabled()) {
```

Comment:
Nested double negative hurts my head. Could you decompose this into 2 methods: like this.clowderDisabled and this.clowderConfigFileMissing and then recast the constructor so it calls them in serial. Something like

```
constructor() {
    if ( this.clowderDisabled ) {
          // message and bail out
    }
    if ( this.clowderFileMissing ) {
        // message and bail out
    }
    // Do the business
}
```